src/Tests/test.runsettings

```diff
@@ -2,7 +2,7 @@
 <RunSettings>
```

Comment:
Let's just delete this file. I don't think setting this environment variable helps at all.